app/internal/international_days/get_international_days.py

```diff
@@ -0,0 +1,17 @@
+from typing import Optional
+
+from sqlalchemy.orm import Session
+from sqlalchemy.sql.expression import func
+
+from app.database.models import InternationalDays
+
+
+def international_day_per_day(session: Session, date) ->\
+        Optional[InternationalDays]:
```

Comment:
The function should probably be a verb and as it returns something, it should probably be "get_international_day" (or "get_international_day_per_day" if the "per day" part is needed).

Reply:
fixed